pipelines/calexpMetrics.yaml

```diff
@@ -1,37 +0,0 @@
-description: |
```

Comment:
Can this be deprecated instead of removed?

Reply:
Yes - I've reinstated calexpMetrics.yaml, added a note to its description, and have created a ticket to remove it eight weeks from now (as per the DM guidelines).